tests/ObjectConfiguration.tex

```diff
@@ -1,6 +1,6 @@
 \section{Replicating a Configuration of Objects}
 
-The robot has to understand relationships between objects, instruct a human to replicate that configuration of objects, and fix errors that the human makes.
+The robot has to understand relationships between objects, instruct a human operator to replicate that configuration of objects, and fix all errors made by the operator.
```

Comment:
Understand is a strong word here. I don't understand why the pots in my mother balcony are in certain order, but I am able to remember and replicate its configuration. However, I do understand why fork and knife are located the way they are in a table setup so I can explain it.